gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-fetch-configs.groovy

```diff
@@ -2,7 +2,7 @@
   String parameter TARGET_ENVIRONMENT
     e.g., qa-anvil
 
-  Archived artifact - gitops.json
+  Archived artifacts - gitops.json, manifest.json
```

Comment:
why are we adding manifest.json to artifacts

Reply:
I see we pull it later

Reply:
i need the dictionary url to run the data simulator to get test data for the "submit and query" tests. I thought might as well get the whole config since we will probably need more values